src/test/java/com/epam/tc/hw3/ex1/CheckLoginPage.java

```diff
@@ -0,0 +1,90 @@
+package com.epam.tc.hw3.ex1;
+
+import com.epam.tc.hw3.AbstractBeforeAfter;
+import com.epam.tc.hw3.driver.utils.Constants;
+import com.epam.tc.hw3.pages.HomePage;
+import com.epam.tc.hw3.pages.IndexPage;
+import java.util.Locale;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.Test;
+
+public class CheckLoginPage extends AbstractBeforeAfter {
+    static final SoftAssertions SOF = new SoftAssertions();
+
+    @Test
+    public void loginPageTest() {
+        //Assert Browser title
+        String titleName = new HomePage(webDriver)
```

Comment:
It would be better if you declared a new HomePage once and then worked with this object, and not creating a new instance for every action with it